src/platform/Kconfig

```diff
@@ -467,7 +467,7 @@ config SOF_STACK_SIZE
 	  components.
 
 config SECONDARY_CORE_DISABLING_TIMEOUT
-	int
+	int "Secondary core disabling timeout"
 	default 5000
```

Comment:
@RanderWang Can you make a patch that changes the default to 400ms instead? SOF driver and firmware are hosted in same public project, so it makes sense to adapt any defaults between the two main entities. Special configs (like for FPGA) should be the exception, not the norm. Otherwise we need to keep doing the same change for all new platforms..

Reply:
@kv2019i  updated, thanks

Reply:
FPGA does need 5000ms, in future, if SOF need run on FPGA again, we'd better to have a mechanism to change it back.